libs/ui/lib/listbox/Listbox.tsx

```diff
@@ -68,11 +83,13 @@ export const Listbox = <Value extends string = string>({
         },
       }),
     ],
+    whileElementsMounted: autoUpdate,
   })
 
   const selectedItem = selected && items.find((i) => i.value === selected)
   const noItems = !isLoading && items.length === 0
   const isDisabled = disabled || noItems
+  const zIndex = getDropdownZIndex(useIsInModal(), useIsInSideModal())
```

Comment:
Since this has just the one call, I'd be fine with inlining this function. You lose the test, but I don't think it's testing much anyway since most of the correctness comes from passing the right two booleans to it.

```ts
const isInModal = useIsInModal()
const isInSideModal = useIsInSideModal()
const zIndex = isInModal
  ? 'z-modalDropdown'
  : isInSideModal
  ? 'z-sideModalDropdown'
  : 'z-contentDropdown'

```

Reply:
Solid. Wasn't sure if nested ternaries were verboten or not, but would also prefer this over a whole other function.